oci.go

```diff
@@ -41,6 +41,11 @@ var (
 var cgroupsMemDirPath = "/sys/fs/cgroup"
 
 func containerExists(containerID string) (bool, error) {
+
```

Comment:
I think this summarises to a nop for this PR? Maybe @dlespiau can raise an issue/PR for any problems with `containerExists()`?